libs/auth-api-lib/src/lib/services/delegations.service.ts

```diff
@@ -419,15 +434,21 @@ export class DelegationsService {
           model: DelegationScope,
           required: true,
           where: this.getScopeValidWhereClause(DelegationValidity.NOW),
+          include: [
+            {
+              model: ApiScope,
+              as: 'apiScope',
+              required: true,
```

Comment:
Is this `required` redundant?

Reply:
Yes actually, this and the one above in L#435.